src/sage/structure/proof/proof.py

```diff
@@ -196,10 +196,12 @@ class WithProof():
     systems for a block of code, with a guarantee that it will be set
     back to how it was before after the block is done, even if there is an error.
 
-    EXAMPLES::
+    EXAMPLES:
+
+    This would hang "forever" if attempted with ``proof=True``::
 
         sage: proof.arithmetic(True)
-        sage: with proof.WithProof('arithmetic',False):    # this would hang "forever" if attempted with proof=True
```

Comment:
Unless this comment is no longer relevant, it should remain.

Reply:
The comment appears in text just above:
https://github.com/sagemath/sage/blob/8199be3f525a104aa5884040c214c94602f33c42/src/sage/structure/proof/proof.py#LL200C1-L208C1